src/definitions/modules/dropdown.less

```diff
@@ -1299,9 +1299,14 @@ select.ui.dropdown {
   border-radius: @floatingMenuBorderRadius !important;
 }
 .ui.floating.dropdown > .menu {
-  margin-top: @floatingMenuDistance !important;
   border-radius: @floatingMenuBorderRadius !important;
 }
+.ui:not(.upward).floating.dropdown > .menu {
+  margin-top: @floatingMenuDistance !important;
```

Comment:
Remove the !important, it's not needed

Reply:
That's how it is on version 2.7.4 : sure you want I change that ?

Reply:
Yes, i tested it before posting and the !important is still inherit from SUI, not a specific 2.7.4 feature.
But of course, i advise you to test it and get convinced yourself.